src/rcpp_octave.cpp

```diff
@@ -219,7 +267,7 @@ void R_unload_RcppOctave(DllInfo *info)
  * @note OCTAVE_API_VERSION_NUMBER is 47 for 3.4.0 but 45 for 3.4.2
  * see: http://octave.1599824.n4.nabble.com/API-version-going-backwards-td3722496.html
  */
-extern void recover_from_exception(void)
```

Comment:
This function name was colliding with a deprecated inline function of the same name in `libinterp/corefcn/interpreter.h`